docassemble/ALWeaver/data/questions/assembly_line.yml

```diff
@@ -806,6 +806,9 @@ code: |
       mako_matches = get_mako_matches(document)
       if mako_matches:
         mako_syntax_in_docx
+      pdf_variable_name_in_docx_matches = get_pdf_variable_name_matches(document)
+      if pdf_variable_name_in_docx_matches:
```

Comment:
If I'm understanding what I'm seeing correctly, a small improvement might be to make this an explicit check of length instead of a truthy check.

```suggestion
      if len(pdf_variable_name_in_docx_matches) > 0:
```